README.md

```diff
@@ -24,10 +24,7 @@ in or follow specific authors.
 
 ## Screenshots
 
-![Screenshot showing For You screen](docs/images/screenshot-1-foryou.png "Screenshot showing For You screen") 
```

Comment:
I decided to leave it in because they're still currently indexed and served to pages, I'll do a follow up CL to remove them once the new screenshots have been indexed. A cron job runs daily for this, so conservative TTL is 36 hrs.